piu/cli.py

```diff
@@ -280,8 +280,8 @@ def request_access(obj, host, reason, reason_cont, user, password, even_url, odd
 
 
 def request_access_interactive():
-    region_name = click.prompt('AWS region', default=os.getenv('PIU_REGION') or
-                               subprocess.getoutput('aws configure get region'))
+    region_name = os.getenv('PIU_REGION') or click.prompt('AWS region',
```

Comment:
ohoh, please check how Senza does it: `AWS_DEFAULT_REGION` and `~/.aws/config` is honored